runtime/lua/vim/lsp/rpc.lua

```diff
@@ -133,7 +133,8 @@ local function request_parser_loop()
   end
 end
 
-local client_errors = vim.tbl_add_reverse_lookup {
+--- Mapping of error codes used by the client
+local client_errors = {
```

Comment:
I wanted to add an annotation here, but it kept it getting squashed with the next section for `format_rpc_error`.

<details>
<summary>expand diff</summary>

```diff
diff --git a/runtime/doc/lsp.txt b/runtime/doc/lsp.txt
index 1b7d727df..7ddad7ca1 100644
--- a/runtime/doc/lsp.txt
+++ b/runtime/doc/lsp.txt
@@ -1825,7 +1825,10 @@ should_log({level})                                 *vim.lsp.log.should_log()*
 Lua module: vim.lsp.rpc                                              *lsp-rpc*
 
 format_rpc_error({err})                       *vim.lsp.rpc.format_rpc_error()*
-                Constructs an error message from an LSP error object.
+                Mapping of error codes used by the client
+
+                @type table<string, number> Constructs an error message from
+                an LSP error object.
 
                 Parameters: ~
                     {err}  (table) The error object
diff --git a/runtime/lua/vim/lsp/rpc.lua b/runtime/lua/vim/lsp/rpc.lua
index 1ecac50df..f36965bce 100644
--- a/runtime/lua/vim/lsp/rpc.lua
+++ b/runtime/lua/vim/lsp/rpc.lua
@@ -134,6 +134,8 @@ local function request_parser_loop()
 end
 
 --- Mapping of error codes used by the client
+---
+---@type table<string, number>
 local client_errors = {
   INVALID_SERVER_MESSAGE       = 1;
   INVALID_SERVER_JSON          = 2;
```
</details>